app/models/setting/rh_cloud.rb

```diff
@@ -20,6 +20,7 @@ def self.default_settings
       set('rh_cloud_token', N_('Authentication token to Red Hat cloud services. Used to authenticate requests to cloud APIs'), nil, N_('Red Hat Cloud token'), nil, encrypted: true),
       set('exclude_installed_packages', N_('Exclude installed packages from being uploaded to the Red Hat cloud'), false, N_("Exclude installed Packages")),
       set('include_parameter_tags', N_('Should import include parameter tags from Foreman?'), false, N_('Include parameters in insights-client reports')),
+      set('rhc_instance_id', N_('RHC daemon id'), nil, N_('ID of the RHC(Yiggdrasil) daemon')),
```

Comment:
It is just `Yggdrasil`. How should I understand this? Will there be multiple instances of RHC running on the Satellite?

Reply:
It's a single instance that runs on the host.

Reply:
We don't know its identifier. It's an ID derived from certificates used for the authentication.
We need this id to pass it on to Sources.